vision/tox.ini

```diff
@@ -7,6 +7,7 @@ localdeps =
     pip install --quiet --upgrade {toxinidir}/../core
 deps =
     {toxinidir}/../core
+    mock
```

Comment:
No. You did the right thing, `mock < pytest` in alphabetical order. I screwed it up elsewhere.